### ISSUE TITLE
SI-UBL 1.2 Invoices incorrectly validated as valid

### ISSUE THREAD

**Issue by ThijsFTW** [phax/phive-rules#9]
When using phive-rules-simplerinvoicing to validate SI-UBL 1.2 files, the files get validated as valid when they shouldn't.

The peppol validator will mark the files **_correctly_** as invalid: https://peppol-test.nl/validate
The ecosio validator will mark the files **_incorrectly_** as valid: https://ecosio.com/en/peppol-and-xml-document-validator/

Files used for validation come from: https://github.com/peppolautoriteit-nl/testset/tree/master/SI-UBL-1.2. The invoices tagged as 'error' are used.

The following files were incorrectly marked as valid:
SI-UBL-INV-1.2-error-CL-T10-R003.xml
SI-UBL-INV-1.2-error-CL-T10-R008.xml
SI-UBL-INV-1.2-error-CODE-R004.xml
SI-UBL-INV-1.2-error-OP-T10-R002.xml
SI-UBL-INV-1.2-error-OP-T10-R003.xml
SI-UBL-INV-1.2-error-OP-T10-R004.xml
SI-UBL-INV-1.2-error-OP-T10-R006.xml
SI-UBL-INV-1.2-error-OP-T10-R008.xml
SI-UBL-INV-1.2-error-SI-INV-R025.xml

**Comment by phax** [comment]
For the record - these files: [si12-failing-testfiles.zip](https://github.com/phax/phive-rules/files/7104854/si12-failing-testfiles.zip)

**Comment by phax** [comment]
Okay, I checked `SI-UBL-INV-1.2-error-CL-T10-R003.xml`

https://peppol-test.nl/validate correctly claims the error `SI-V12-INV-R026` which is denoted as:
```xml
    <rule context="cbc:*/@currencyID" flag="fatal">
        <assert test="string(.) = string(/ubl:Invoice/cbc:DocumentCurrencyCode)" flag="fatal">[SI-V12-INV-R026]-Currency Identifier MUST be stated in the currency as defined on header level</assert>
    </rule>
```

the created SVRL by phive is this, where you can see that `@currencyID` is never part of the fired-rules

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<schematron-output xmlns="http://purl.oclc.org/dsdl/svrl" schemaVersion="" title="Simplerinvoicing invoice v1.2 bound to UBL 2.1 and OPENPEPPOL v2">
<!--   
		   
		   
		 -->
  <ns-prefix-in-attribute-values prefix="cbc" uri="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" />
  <ns-prefix-in-attribute-values prefix="cac" uri="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" />
  <ns-prefix-in-attribute-values prefix="ubl" uri="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" />
  <ns-prefix-in-attribute-values prefix="xs" uri="http://www.w3.org/2001/XMLSchema" />
  <active-pattern document="" id="SI-UBL-VERSIONS" name="SI-UBL-VERSIONS" />
  <fired-rule context="/" />
  <fired-rule context="ubl:Invoice" />
  <fired-rule context="cbc:CustomizationID" />
  <active-pattern document="" id="SI-V12-INV-WARNING" name="SI-V12-INV-WARNING" />
  <fired-rule context="ubl:Invoice" />
  <fired-rule context="*" />
  <fired-rule context="cbc:CustomizationID" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:InvoicePeriod" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:AccountingSupplierParty" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:PostalAddress | cac:Address" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:AccountingCustomerParty" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:PostalAddress | cac:Address" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:Delivery" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:PostalAddress | cac:Address" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:PaymentMeans" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:PaymentTerms" />
  <fired-rule context="*" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="cac:TaxTotal" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <fired-rule context="*" />
  <active-pattern document="" id="SI-V12-INV" name="SI-V12-INV" />
  <fired-rule context="/" />
  <fired-rule context="ubl:Invoice" />
  <fired-rule context="cbc:CustomizationID" />
  <fired-rule context="cac:Country | cac:OriginCountry" />
  <fired-rule context="cbc:CompanyID" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:PartyLegalEntity" />
  <fired-rule context="cbc:CompanyID" />
  <fired-rule context="cac:Country | cac:OriginCountry" />
  <fired-rule context="cbc:CompanyID" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:PartyLegalEntity" />
  <fired-rule context="cbc:CompanyID" />
  <fired-rule context="cac:Country | cac:OriginCountry" />
  <fired-rule context="cac:PayeeFinancialAccount" />
  <fired-rule context="cbc:ID[parent::cac:PayeeFinancialAccount]" />
  <fired-rule context="cac:FinancialInstitution" />
  <fired-rule context="cbc:ID[parent::cac:FinancialInstitution]" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="ubl:Invoice/cac:TaxTotal[cac:TaxSubtotal/cac:TaxCategory/cac:TaxScheme/cbc:ID = 'VAT']/cbc:TaxAmount" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:LegalMonetaryTotal" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="cac:Item" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="cac:Item" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:InvoiceLine" />
  <fired-rule context="cac:AllowanceCharge" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:TaxCategory" />
  <fired-rule context="cac:TaxScheme" />
  <fired-rule context="cac:Item" />
  <fired-rule context="cac:TaxScheme" />
  <active-pattern document="" id="SI-V12-CODES" name="SI-V12-CODES" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <fired-rule context="cbc:ID[parent::cac:TaxScheme]" />
  <active-pattern document="" id="BIIRULES-UBL-T10" name="BIIRULES-UBL-T10" />
  <fired-rule context="/ubl:Invoice" />
  <fired-rule context="//cac:InvoicePeriod" />
  <fired-rule context="//cac:PaymentMeans" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="//cac:TaxSubtotal[cac:TaxCategory/cac:TaxScheme/cbc:ID = 'VAT']" />
  <fired-rule context="//cac:TaxSubtotal[cac:TaxCategory/cac:TaxScheme/cbc:ID = 'VAT']" />
  <fired-rule context="//cac:LegalMonetaryTotal" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:TaxSubtotal[cac:TaxCategory/cac:TaxScheme/cbc:ID = 'VAT']" />
  <active-pattern document="" id="CodesT10" name="CodesT10" />
  <fired-rule context="cbc:InvoiceTypeCode" />
  <fired-rule context="cbc:DocumentCurrencyCode" />
  <fired-rule context="cac:Country//cbc:IdentificationCode" />
  <fired-rule context="cac:Country//cbc:IdentificationCode" />
  <fired-rule context="cac:Country//cbc:IdentificationCode" />
  <fired-rule context="cac:PaymentMeans//cbc:PaymentMeansCode" />
  <fired-rule context="cac:AllowanceCharge//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:AllowanceCharge//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:AllowanceCharge//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <fired-rule context="cac:TaxCategory/cbc:ID" />
  <active-pattern document="" id="PEPPOL12-UBL-T10" name="PEPPOL12-UBL-T10" />
  <fired-rule context="/ubl:Invoice" />
  <fired-rule context="//cbc:InvoiceTypeCode" />
  <fired-rule context="//cac:AccountingSupplierParty" />
  <fired-rule context="//cac:Country/cbc:IdentificationCode" />
  <fired-rule context="//cac:PartyLegalEntity/cbc:CompanyID" />
  <fired-rule context="//cac:AccountingCustomerParty" />
  <fired-rule context="//cac:Country/cbc:IdentificationCode" />
  <fired-rule context="//cac:PartyLegalEntity/cbc:CompanyID" />
  <fired-rule context="//cac:Country/cbc:IdentificationCode" />
  <fired-rule context="//cac:PaymentMeans" />
  <fired-rule context="//cbc:PaymentMeansCode" />
  <fired-rule context="//cac:PayeeFinancialAccount/cbc:ID" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="/ubl:Invoice/cac:AllowanceCharge" />
  <fired-rule context="//cbc:AllowanceChargeReasonCode" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="/ubl:Invoice/cac:TaxTotal" />
  <fired-rule context="/ubl:Invoice/cac:TaxTotal/cac:TaxSubtotal" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="/ubl:Invoice/cac:TaxTotal/cac:TaxSubtotal" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:LegalMonetaryTotal/child::*" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:InvoiceLine" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <fired-rule context="//cac:TaxCategory" />
  <fired-rule context="//cac:TaxCategory/cbc:ID" />
  <active-pattern document="" id="CodesT10" name="CodesT10" />
</schematron-output>
```

This boils down to a very simple problem in XSLT when it comes to selecting attributes:

Source: https://stackoverflow.com/a/28305335/15254
> The problem with your current approach is that the template matching the attribute/s is never applied. It isn't applied, because the built-in template rule matches only the root node and element nodes, and it applies templates only to children nodes. To apply a template to attributes, you must apply it explicitly yourself.

So I would say the rules are badly designed when you want an XSLT2 engine to apply it.

My proposal would be to change the rules as follows (untested):
```xml
    <rule context="cbc:*[@currencyID]" flag="fatal">
        <assert test="string(@currencyID) = string(/ubl:Invoice/cbc:DocumentCurrencyCode)" flag="fatal">[SI-V12-INV-R026]-Currency Identifier MUST be stated in the currency as defined on header level</assert>
    </rule>
```

hth

**Comment by phax** [comment]
With the last testset update ~3 hours ago, the following `error` files are not failing for SI 1.2 on my side:
* SI-UBL-INV-1.2-error-classifiedtaxcat-no-id.xml
* SI-UBL-INV-1.2-error-CL-T10-R003.xml
* SI-UBL-INV-1.2-error-CL-T10-R008.xml
* SI-UBL-INV-1.2-error-CL-T10-R010.xml
* SI-UBL-INV-1.2-error-CODE-R004.xml
* SI-UBL-INV-1.2-error-corrective-billingreference.xml
* SI-UBL-INV-1.2-error-OP-T10-R002.xml
* SI-UBL-INV-1.2-error-OP-T10-R003.xml
* SI-UBL-INV-1.2-error-OP-T10-R004.xml
* SI-UBL-INV-1.2-error-OP-T10-R006.xml
* SI-UBL-INV-1.2-error-OP-T10-R008.xml
* SI-UBL-INV-1.2-error-SI-INV-R025.xml

Orders:
* SI-UBL-PO-1.2-error-BII2-T01-R015.xml
* SI-UBL-PO-1.2-error-SI-V12-PO-R200.xml

**Comment by ThijsFTW** [comment]
> With the last testset update ~3 hours ago, the following `error` files are not failing for SI 1.2 on my side:
> 
> * SI-UBL-INV-1.2-error-classifiedtaxcat-no-id.xml
> * SI-UBL-INV-1.2-error-CL-T10-R003.xml
> * SI-UBL-INV-1.2-error-CL-T10-R008.xml
> * SI-UBL-INV-1.2-error-CL-T10-R010.xml
> * SI-UBL-INV-1.2-error-CODE-R004.xml
> * SI-UBL-INV-1.2-error-corrective-billingreference.xml
> * SI-UBL-INV-1.2-error-OP-T10-R002.xml
> * SI-UBL-INV-1.2-error-OP-T10-R003.xml
> * SI-UBL-INV-1.2-error-OP-T10-R004.xml
> * SI-UBL-INV-1.2-error-OP-T10-R006.xml
> * SI-UBL-INV-1.2-error-OP-T10-R008.xml
> * SI-UBL-INV-1.2-error-SI-INV-R025.xml
> 
> Orders:
> 
> * SI-UBL-PO-1.2-error-BII2-T01-R015.xml
> * SI-UBL-PO-1.2-error-SI-V12-PO-R200.xml

That's after the applying the fix you've proposed? Because the updated test files are still failing for me with version 2.1.7.

**Comment by phax** [comment]
I didn't change the rules - this is the job of the SimplerInvoicing guys ;)
I'd expect that this would result in a 1.2.4 release on their side. And it should be evaluated for similar cases.

To me it seems to be a "flaw" of the xslt engine they are using at their side.... I will create a ticket and link to this issue

**Comment by phax** [comment]
It seems like I forgot it - shame on me. Now it is done...

**Comment by ri4a** [comment]
Let's just deprecate SI-1.2. It's the only Peppol BIS V2 based format left on the earth.

**Comment by phax** [comment]
I think it is still in use for Orders if I remember correctly

**Comment by ri4a** [comment]
Ah, yes, I should have said

```
the last Peppol BIS V2 _Invoice_ based format left on earth.
```

But let's deprecate SI-1.2 Order too ;).

Other active ":ver2.0:" aside from SI-1.2 Invoice/Order:

```
urn:oasis:names:specification:ubl:schema:xsd:Order-2::Order##urn:www.cenbii.eu:transaction:biitrns001:ver2.0:extended:urn:www.peppol.eu:bis:peppol28a:ver1.0:extended:urn:fdc:peppol-authority.co.uk:spec:ordering:ver1.0::2.1
```
and

```
urn:oasis:names:specification:ubl:schema:xsd:OrderResponse-2::Order##urn:www.cenbii.eu:transaction:biitrns076:ver2.0:extended:urn:www.peppol.eu:bis:peppol28a:ver1.0:extended:urn:fdc:peppol-authority.co.uk:spec:ordering:ver1.0::2.1
```
on which I have no opinion except they look old.

**Comment by phax** [comment]
@ri4a  I guess that is a wonderful suggestion to https://github.com/peppolautoriteit-nl/validation/issues :)

**Comment by phax** [comment]
This has been fixed with SI 1.2.4, included in the May 2022 release of SimplerInvoicing which was incorporated into phive-rules 2.1.13 release.
